content/javascript/concepts/operators/operators.md

```diff
@@ -175,3 +175,64 @@ let isDuck = (walksLikeADuck && talksLikeADuck) ? "It's a duck" : "It's not a du
 
 console.log(isDuck);
 ```
+
+The ternary operator can also perform multiple steps based on a certain condition, similar to the IF/ELSE condition statement.
+
+For example, the following IF/ELSE statement...
```

Comment:
```suggestion
For example, the following `if...else` statement...
```

Reply:
Thank you @SSwiniarski for letting me know about these changes! I'm uploading them now.